README

```diff
@@ -4,26 +4,26 @@ This is version 10.0.0 of WiredTiger.
 
 WiredTiger release packages and documentation can be found at:
 
-	http://source.wiredtiger.com/
+	https://source.wiredtiger.com/
 
 The documentation for this specific release can be found at:
 
-	http://source.wiredtiger.com/10.0.0/index.html
+	https://source.wiredtiger.com/develop/index.html
```

Comment:
Sorry, I just meant `http => https`. We still want to list `10.0.0` here.
I realize the link doesn't exist but that's because the release hasn't gone out yet.